static/app/views/issueList/issueViews/issueViewsList/issueViewsList.tsx

```diff
@@ -258,106 +259,110 @@ export default function IssueViewsList() {
   }
 
   return (
-    <Layout.Page>
-      <Layout.Header unified>
-        <Layout.HeaderContent>
-          <Layout.Title>{t('All Views')}</Layout.Title>
-        </Layout.HeaderContent>
-        <Layout.HeaderActions>
-          <ButtonBar gap={1}>
-            {openFeedbackForm ? (
+    <SentryDocumentTitle title={t('All Views')} orgSlug={organization.slug}>
```

Comment:
All I did here was wrap the entire component in `<SentryDocumentTitle>`, no other changes